changelogs/fragments/55903_kubevirt.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
```

Shouldn't that be:
```suggestion
minor_changes:
```
? I don't see how this is a bugfix.

---

This parameter was supposed to be `interface_name`. I am waiting for Kubevirt maintainers to comment and will update accordingly.